Gibe.DittoProcessors/Processors/AbsoluteUrlAttribute.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Umbraco.Core.Models;
+using Umbraco.Web;
+
+namespace Gibe.DittoProcessors.Processors
+{
+	public class AbsoluteUrlAttribute : TestableDittoProcessorAttribute
+	{
+
+		public override object ProcessValue()
+		{
+			var content = Value as IPublishedContent;
```

Comment:
Perhaps `(Value as IPublishedContent) ?? Context.Content`? Though if we're making changes here, perhaps we could put something much like that into one of the supertypes here to make it more likely we can use that logic wherever valuable.